warmup.py

```diff
@@ -240,17 +247,42 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if (self.rmax > (L / 2)):
+            raise ValueError('rmax > L/2')
+        else:
```

Comment:
You don't need to wrap this whole code block in an `else`, since it just adds unnecessary documentation. In python, you can assume the code block terminates after the exception is raised (so the rest of this is never reached).